README.md

```diff
@@ -1,29 +1,200 @@
 # Validations with `form_tag`
 
-## Objectives
+Now that we've learned to handle the server side of validations, we need to
+take care of the client side.
+
+At this point, we'll be in step three of the following flow:
+
+1. User fills out the form and hits "Submit", transmitting the form data via
+   a POST request.
+2. The controller sees that validations have failed, and re-renders the form.
+3. **The view displays the errors to the user**.
+
+# Objectives
+
+After this lesson, you'll be able to...
 
-- render or redirect based on validation of instance in create/update
 - Prefill in form values based on an instance
 - Print out full error messages based on an invalid instance
-- introspect on errors for a field
-- apply an error class to a field conditionally based on errors on a field
+- Introspect on errors for a field
+- Apply an error class to invalid fields
+
+
+# Pre-Filling Form Values
```

Comment:
I like how you explain why you want to pre-fill the forms. But at this point I'm a bit confused about what we're pre-filling the fields with. Is this a POST or a PUTS/PATCH request? Is this form for editing this object?